pkg/modules/modules.go

```diff
@@ -10,12 +10,20 @@ SPDX-License-Identifier: Apache-2.0
 package modules
 
 import (
+	"github.com/filecoin-project/mir/pkg/pb/statuspb"
 	t "github.com/filecoin-project/mir/pkg/types"
 )
 
 // Module generalizes the ActiveModule and PassiveModule types.
 type Module interface {
+
+	// ImplementsModule only serves the purpose of indicating that this is a Module and must not be called.
 	ImplementsModule()
+
+	// Status returns the current state of the module.
+	// Mostly for debugging purposes.
+	// TODO: This functionality is not yet implemented and all the Status() implementations are stubs. Fix that.
+	Status() (s *statuspb.ProtocolStatus, err error)
```

Comment:
This method is never used nor implemented in the current code. Maybe just remove it for now?

Reply:
In fact I've been hesitating to do that for some time already and kept it around because it has been part of the old design. But there is no rational reason to keep it at this point, we'll re-introduce it when we are ready to implement it. Will remove.